region/crateria/east/The Moat.json

```diff
@@ -473,91 +455,68 @@
           "minTiles": 3
         }
       },
-      "requires": []
-    },
-    {
-      "link": [2, 1],
-      "name": "Pass Below",
-      "requires": [
-        "h_canBombThings"
-      ]
+      "requires": [],
+      "collectsItems": [3]
     },
     {
-      "link": [2, 1],
-      "name": "Leave With Spark (Come In Shinecharged)",
+      "link": [1, 3],
+      "name": "Carry Shinecharge to Item",
       "entranceCondition": {
         "comeInShinecharged": {
-          "framesRequired": 90
+          "framesRequired": 85
         }
       },
-      "requires": [
-        "canShinechargeMovementComplex",
-        {"shinespark": {"frames": 23}}
-      ],
-      "exitCondition": {
-        "leaveWithSpark": {}
-      },
-      "unlocksDoors": [
-        {"types": ["super"], "requires": []},
-        {"types": ["missiles", "powerbomb"], "requires": ["never"]}
-      ],
-      "flashSuitChecked": true,
-      "note": [
-        "Come into the room with a shinecharge, run and jump to the left, firing a shot just before landing on the pedestal.",
-        "Activate the shinespark either just before landing on the pedestal or just afterward (depending on the vertical position needed for the spark)."
-      ]
-    },
-    {
-      "link": [2, 1],
-      "name": "Leave With Spark (Come In Shinecharging)",
-      "entranceCondition": {
-        "comeInShinecharging": {
-          "length": 4,
-          "openEnd": 0
-        }
-      },
-      "requires": [
-        "canShinechargeMovementComplex",
-        {"shinespark": {"frames": 23}}
-      ],
-      "exitCondition": {
-        "leaveWithSpark": {}
-      },
-      "unlocksDoors": [
-        {"types": ["super"], "requires": []},
-        {"types": ["missiles", "powerbomb"], "requires": ["never"]}
-      ],
-      "note": [
-        "After gaining a shinecharge, run back to the door to make space for a running jump to the left, firing a shot just before landing on the pedestal.",
-        "Activate the shinespark either just before landing on the pedestal or just afterward (depending on the vertical position needed for the spark)."
-      ]
+      "requires": [],
+      "collectsItems": [3],
+      "endsWithShineCharge": true
     },
     {
-      "link": [2, 1],
-      "name": "Hero Shot Spark",
+      "link": [1, 3],
+      "name": "Hero Shot Spark Through Item",
       "entranceCondition": {
         "comeInShinecharged": {
-          "framesRequired": 25
+          "framesRequired": 40
         }
       },
       "requires": [
         "canShinechargeMovementTricky",
         "canHeroShot",
-        {"shinespark": {"frames": 39}}
+        {"shinespark": {"frames": 38}}
       ],
-      "exitCondition": {
-        "leaveWithSpark": {}
-      },
       "unlocksDoors": [
         {"types": ["super"], "requires": []},
         {"types": ["missiles", "powerbomb"], "requires": ["never"]}
       ],
```

Comment:
I dont think this is needed here? if it is, it needs to specify it's to unlock door 2
```suggestion
```